test/v1/reliefInvestments/retrieve/def1/Def1_RetrieveForeignReliefsValidatorFactorySpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.reliefInvestments.retrieve.def1
+
+import api.models.domain.{Nino, TaxYear}
+import api.models.errors._
+import support.UnitSpec
+import v1.reliefInvestments.retrieve.RetrieveReliefInvestmentsValidatorFactory
+import v1.reliefInvestments.retrieve.def1.model.request.Def1_RetrieveReliefInvestmentsRequestData
+import v1.reliefInvestments.retrieve.model.request.RetrieveReliefInvestmentsRequestData
+
+class Def1_RetrieveForeignReliefsValidatorFactorySpec extends UnitSpec {
```

Comment:
Ah you've used the foreign reliefs to complete this PR, so please rename the class back to validator (and remove the foreign bit as well).